src/app/room/components/room.component.html

```diff
@@ -1,3 +1,5 @@
+<p class="text-container">Let's find out who's gonna be the final  {{(room$ | async)?.typeOfGame === 1 ? 'loser' : 'winner'}}</p>
```

Comment:
There is two spaces here. 
Might I also suggest something more obvious or funny instead of the one word that is different?
`the last one will be the glorious winner!`
`the last one will be the loser and never life out his shame!`
or something?